### DISCUSSION TITLE
WS_POPUPWINDOW and WM_NCHITTEST message

### DISCUSSION THREAD

**Discussion by jb-perrier** [rust-windowing/winit#3013] [Q&A] [answered]
Pretty sure i'm not the first one to bring the subject here, but i can't find any dedicated question.

Is there any support for the `WS_POPUPWINDOW` style and user handling of `WM_NCHITTEST` message ?
I need it in order to fully customize the caption bar.

**Comment by madsmtm** [comment, marked as answer]
There have been some talk in the past about allowing users to override those events themselves: https://github.com/rust-windowing/winit/issues/2120, but that's all I know
